### DISCUSSION TITLE
Setting the initial state of SignalStore when Jest testing

### DISCUSSION THREAD

**Discussion by Cabinet20** [ngrx/platform#4253] [Q&A]
I was wondering if there is a way to set the initial state with the new @ngrx/signals.

It was possible to do this in ngrx store like this, in the past.

```provideMockStore({ initialState })```

Thanks in advance for any help!

**Comment by rainerhahnekamp** [comment]
`provideMockStore` is about mocking the `Store` service itself. To my understanding there is no functionality there yet, but might be a good feature to add...

**Comment by rainerhahnekamp** [comment]
@Cabinet20: https://github.com/ngrx/platform/issues/4256
